emails/views.py

```diff
@@ -932,12 +942,13 @@ def _get_text_html_attachments(message_json):
         # assume email content in S3
         bucket, object_key = _get_bucket_and_key_from_s3_json(message_json)
         message_content = get_message_content_from_s3(bucket, object_key)
-        histogram_if_enabled("relayed_email.size", len(message_content))
-
+    email_size = len(message_content)
+    histogram_if_enabled("relayed_email.size", email_size)
     bytes_email_message = message_from_bytes(message_content, policy=policy.default)
 
     text_content, html_content, attachments = _get_all_contents(bytes_email_message)
-    return text_content, html_content, attachments
+    # TODO add logs on the entire size of the email and the time it takes to download/process
```

Comment:
Note: I asked for this `TODO`. We have a metric for how long it took to get the content from S3, and a histogram for the size, but we'd need to log both together to model how long it takes to download from S3 based on the object size. We're also curious if message size is important for email processing in general. It seemed out of scope for this hotfix, so we're deferring it.